pre/main.ipynb

```diff
@@ -82,6 +82,20 @@
     "        return bspump.mqtt.MQTTConnection(app, \"MQTTConnection\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bae448a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@register_connection\n",
+    "def connection(app):\n",
+    "    return bspump.http.web.server.WebServerConnection(app, \"WebServerConnection\", config={\n",
```

Comment:
Why not use the default connection that is created by `jupyter.py`? https://github.com/bitswan-space/BitSwan/blob/24b4a492b80d93ca97bea0adaed14f5e604653a7/bspump/jupyter/jupyter.py#L498

Reply:
I don't understand this comment since I used `@register_connection`?